app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,7 @@
 <resources>
     <!-- Common -->
-    <string name="time_period">%1$s - %2$s%3$s</string>
+    <string name="time_period">%1$s - %2$s</string>
+    <string name="room_format">%1$s%2$s</string>
```

Comment:
If this is not translatable, Please add `translatable="false"`
http://tools.android.com/recent/non-translatablestrings

Reply:
I didn't know this attribute, thank you 🙇  Added https://github.com/DroidKaigi/conference-app-2018/pull/89/commits/b92d287a312d7e7d541ed0760414f1f74fe6e84b.